src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -24,8 +26,10 @@ export class FilterOption extends HTMLElement {
     this.filterOptionTitle = filterContainer.querySelector('.title');
     this.filterOptionTitle.textContent = this.textContent;
 
-    this.checkboxList = filterContainer.querySelector('.checkbox-list');
-    this.checkboxList.classList.add(this.filterKey);
+    this.dropDownList = filterContainer.querySelector(
```

Comment:
Why did you change the name? `checkboxList` is a better description of the functionality. The fact that it looks like a dropdown is not relevant.